### DISCUSSION TITLE
Separate branches in monorepo

### DISCUSSION THREAD

**Discussion by ainvd** [git-lfs/git-lfs#5211] [Q&A] [answered]
Hi,
in my organisation we have chosen to go with a monorepo setup for our repository. Aside from master, we have a few long-lived braches (protected branches) that we work on for a longer period of time.
Is there any way that we can treat the lfs files in the protected branches as a separate project, so that locking a file in a branch created from a protected branch does not lock the file in another protected branch (or master)?

**Comment by bk2204** [comment, marked as answer]
Hey,

The client doesn't offer this capability, but technically it doesn't need to.  The server has all the information it needs to successfully implement this functionality, although I'm not aware of any that do.  The best thing to do would be to ask your server implementer if this is a thing they have.  I know from personal experience that GitHub does not offer this.

**Reply by ainvd** [reply]
I had assumed that some kind of property would have to be implemented in lfs to support this, but I take it from your answer that that is not the case.
I will contact my server implementer and see if it is possible to add support for this. Thank you for your answer.
